libs/dh/metering-point/feature-overview/src/lib/primary-master-data/dh-metering-point-primary-master-data.component.html

```diff
@@ -0,0 +1,101 @@
+Copyright 2021 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container
+  *transloco="let transloco; read: 'meteringPoint.overview.primaryMasterData'"
+>
+  <div class="column flex">
+    <watt-icon
+      name="map_marker"
+      label="map_marker"
+      [size]="iconSizes.XLarge"
+      aria-hidden
+    ></watt-icon>
+    <section class="align-top">
+      <dl>
+        <dt class="watt-label">{{ transloco('address') }}</dt>
+        <dd [innerHTML]="address"></dd>
+      </dl>
+
+      <dl class="multiple-terms-and-descriptions">
+        <ng-container *ngIf="primaryMasterData?.isActualAddress">
+          <dt class="with-icon" [title]="transloco('actualAddress')">
+            <watt-icon
+              name="success"
+              label="success"
+              [size]="iconSizes.XSmall"
+            ></watt-icon>
+          </dt>
+          <dd class="watt-text-xs">{{ transloco('actualAddress') }}</dd>
+        </ng-container>
+
+        <dt class="watt-text-xs">{{ transloco('locationDescription') }}:</dt>
+        <dd class="watt-text-xs">
+          {{ primaryMasterData?.locationDescription ?? fallbackValue }}
+        </dd>
+        <dt class="watt-text-xs text-dimmed">
+          {{ transloco('geoInfoReference') }}:
+        </dt>
+        <dd class="watt-text-xs text-dimmed">
+          {{ primaryMasterData?.darReference || fallbackValue }}
+        </dd>
+      </dl>
+    </section>
+  </div>
+
+  <div class="column">
+    <section>
+      <watt-icon
+        name="power"
+        label="power"
+        [size]="iconSizes.XLarge"
+        aria-hidden
+      ></watt-icon>
+      <dl>
+        <dt class="watt-label">
+          {{ transloco('hasElectricitySupplier') }}
+        </dt>
+        <dd>
+          {{
+            primaryMasterData?.supplyStart ? transloco('yes') : transloco('no')
+          }}
+        </dd>
+        <dd class="watt-text-xs text-dimmed full-width">
+          <ng-container *ngIf="primaryMasterData?.supplyStart; else emptyValue">
+            {{ transloco('since', { date: primaryMasterData?.supplyStart }) }}
+          </ng-container>
+        </dd>
+      </dl>
+    </section>
+
+    <section>
+      <watt-icon
+        name="meter"
+        label="meter"
+        [size]="iconSizes.XLarge"
+        aria-hidden
+      ></watt-icon>
+      <dl>
+        <dt class="watt-label">{{ transloco('meterNumber') }}</dt>
+        <dd>
+          <small>{{ primaryMasterData?.meterId ?? fallbackValue }}</small>
```

Comment:
question: If `primaryMasterData.meterId` was an empty string, should we still show the `fallbackValue`? It will not currently.

Reply:
Is now handled by custom pipe which can be moved to shared lib in our other story